database-management/src/main/resources/db/changelog/2023/create-ab2d-analyst.sql

```diff
@@ -0,0 +1,11 @@
+------------------------------
+-- Create ab2d_analyst user
+------------------------------
+DO
+$$
+begin
+        if not exists(SELECT * FROM pg_user WHERE usename = 'ab2d_analyst') THEN
```

Comment:
```suggestion
        if not exists(SELECT * FROM pg_user WHERE username = 'ab2d_analyst') THEN
```

Reply:
<img width="590" alt="image" src="https://github.com/CMSgov/ab2d-lambdas/assets/121067659/4e394969-e027-4caa-b36e-67e069570a34">

its not a typo